src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -318,4 +318,43 @@ public static void UnknownUrl_Throws_ProtocolException()
             Assert.True(exception.Message.Contains(protocolExceptionUri), string.Format("Expected exception message to contain '{0}'", protocolExceptionUri));
         }
     }
+
+    [Fact]
+    [ActiveIssue(194)]
+    [OuterLoop]
+    public static void DuplexCallback_Throws_FaultException()
+    {
+        DuplexChannelFactory<IWcfDuplexTaskReturnService> factory = null;
+        Guid guid = Guid.NewGuid();
+
+        NetTcpBinding binding = new NetTcpBinding();
+        binding.Security.Mode = SecurityMode.None;
+
+        DuplexTaskReturnServiceCallback callbackService = new DuplexTaskReturnServiceCallback();
+        InstanceContext context = new InstanceContext(callbackService);
+
+        try
+        {
+            var exception = Assert.Throws<FaultException<FaultDetail>>(() =>
+            {
+                using (factory = new DuplexChannelFactory<IWcfDuplexTaskReturnService>(context, binding, new EndpointAddress(Endpoints.Tcp_NoSecurity_TaskReturn_Address)))
+                {
+                    IWcfDuplexTaskReturnService serviceProxy = factory.CreateChannel();
+
+                    Task<Guid> task = serviceProxy.FaultPing(guid);
+                    if ((task as IAsyncResult).AsyncWaitHandle.WaitOne(new TimeSpan(0, 0, 10)))
```

Comment:
You've used a magic number for the timeout. Please replace with a named constant.